demo-shell/src/app/components/app-layout/cloud/people-groups-cloud.component.ts

```diff
@@ -0,0 +1,95 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, ViewEncapsulation } from '@angular/core';
+import { MatCheckboxChange } from '@angular/material';
+import { PeopleCloudComponent, GroupCloudComponent, GroupModel } from '@alfresco/adf-process-services-cloud';
+
+@Component({
+    selector: 'app-people-groups-cloud',
+    templateUrl: './people-groups-cloud.component.html',
+    styleUrls: ['./people-groups-cloud.component.scss'],
+    encapsulation: ViewEncapsulation.None
+})
+export class PeopleGroupCloudComponent {
+
+    people: any = {
+        mode: PeopleCloudComponent.MODE_SINGLE,
```

Comment:
can we have flat properties instead of having an object with subproperties? 
i.e.
peopleMode
preSelectUser

groupMode
preSelectGroup